torch_xla/csrc/tensor.cpp

```diff
@@ -1828,10 +1828,22 @@ bool XLATensor::ShouldSyncIrNode() {
   return this->data()->ir_value->op() != xla_device_data;
 }
 
+c10::SymIntNode XLASymIntNodeImpl::add(const c10::SymIntNode& other) {
+  auto pother = dynamic_cast<XLASymIntNodeImpl*>(other.get());
+  auto nadd = torch::lazy::MakeNode<SizeAdd>(node(), pother->node());
+  return c10::make_intrusive<XLASymIntNodeImpl>(nadd);
+}
+
 c10::SymIntNode XLASymIntNodeImpl::mul(const c10::SymIntNode& other) {
   auto pother = dynamic_cast<XLASymIntNodeImpl*>(other.get());
   auto nmul = torch::lazy::MakeNode<torch_xla::SizeMul>(node(), pother->node());
   return c10::make_intrusive<XLASymIntNodeImpl>(nmul);
 }
 
+c10::SymIntNode XLASymIntNodeImpl::floordiv(const c10::SymIntNode& other) {
```

Comment:
I don't think we can do that yet because our SizeDiv only does floor division: https://github.com/pytorch/xla/blob/b8268ffd6bb2c53275e350293aa47b84615f251f/torch_xla/csrc/ops/dynamic_ir.cpp#L142.

On another thought, do you think we should split our SizeDiv into 2 classes: SizeFloorDiv and SizeTrueDiv? @miladm @JackCaoG Though, since SizeDiv represent a size, I'm not sure what a float type size mean.

Reply:
Hm... I can't think of a use case for `truediv`. We can revisit this when the need presents itself.